### ISSUE TITLE
apply_model run time

### ISSUE THREAD

**Issue by mathpopo** [facebookresearch/demucs#561]
i try to use separate(htdemuc),but i meet :  i put 100ms / 1s ,apply_model run time is same,how can i do to see different?

**Comment by CarlGao4** [comment]
What system are you using? What is your CPU and GPU?

**Comment by mathpopo** [comment]
GPU, other issue:all model point:
stereo，44100？
i put a mono 16000, but ->torch audio load :  stereo，44100

**Comment by mathpopo** [comment]
padded_mix = mix.padded(valid_length), should here again enlange the length

**Comment by CarlGao4** [comment]
What? Are you TRAINING demucs?

**Comment by mathpopo** [comment]
@CarlGao4 no, now i just test , debug all struct, understand how toadd log
